src/main/java/seedu/address/model/student/Student.java

```diff
@@ -89,6 +90,14 @@ public boolean isSameStudentId(StudentId otherStudentId) {
                 && otherStudentId.equals(getStudentId());
     }
 
+    public boolean hasCompletedAssignment() {
+        return hasCompletedAssignment;
```

Comment:
Recommended to add test cases for the components not in UI

Reply:
Done!